server.go

```diff
@@ -67,7 +68,9 @@ func (svr *Server) getHandle(handle string) (*os.File, bool) {
 
 type serverRespondablePacket interface {
 	encoding.BinaryUnmarshaler
+	id() uint32
 	respond(svr *Server) error
+	writing() bool
```

Comment:
Keeping the zero value is useful, but the name of the method isnt very
informative.

On Sat, 9 Jan 2016, 04:21 Matt Layher <notifications@github.com> wrote:

> In server.go <https://github.com/pkg/sftp/pull/78#discussion_r49213108>:
>
> >  	respond(svr *Server) error
> > +	writing() bool
>
> Yeah, my intent was to make the zero-value the "read-only" one, but I
> suppose I can invert it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pkg/sftp/pull/78/files#r49213108>.
>